include/bl_storage.h

```diff
@@ -106,6 +149,68 @@ uint16_t get_monotonic_counter(void);
  */
 int set_monotonic_counter(uint16_t new_counter);
 
+/**
+ * @brief The PSA life cycle states a device can be in.
+ *
+ * The LCS can be only transitioned in the order they are defined here.
+ */
+enum lcs {
+	UNKNOWN = 0,
+	ASSEMBLY = 1,
+	PROVISION = 2,
```

Comment:
Some inconsistencies between this enum, struct life_cycle_state_data and PSA spec.
* provision <-> provisioning
* secure <-> secured

Should the symbols here be prefixed with something? A tleast "UNKNOWN" is likely to crash with something

Reply:
Good point gonna prefix it with LCS